components/TabMenu/TabMenu.js

```diff
@@ -12,26 +12,91 @@ class TabMenu extends PureComponent {
     super(props);
 
     autoBindReact(this);
+
+    this.state = {
+      itemWidths: {},
+      scrollOffset: 0,
+    };
   }
 
-  handleOptionSelected(option) {
+  handleOptionSelected(key, option) {
```

Comment:
For the sake of consistency, shouldn't `key` be after `option` like in the method later on? Not a change request, just food for thought if it makes sense to you.